### DISCUSSION TITLE
Could EP fix the issue with menubar in File Explorer in Win11 22H2 and 23H2 randomly appearing and disappearing?

### DISCUSSION THREAD

**Discussion by Anixx** [valinet/ExplorerPatcher#2994] [Ideas]
https://winclassic.net/thread/1622/menu-bar-disappears-seconds

**Comment by pyrates999** [comment]
Please provide more details and not just a link.

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Amrsatrio** [comment]
@pyrates999 No need. This issue has been present since Tabs were introduced to 22621, and sadly I haven't had the time yet to see the mess they've done to explorerframe.dll.
